src/main/java/com/vinylteam/vinyl/exception/RecoveryPasswordException.java

```diff
@@ -0,0 +1,7 @@
+package com.vinylteam.vinyl.exception;
+
+public class RecoveryPasswordException extends RuntimeException {
+    public RecoveryPasswordException(String message) {
+        super(message);
+    }
+}
```

Comment:
Please add empty row after class declaration and before closing according to project code style. Thanks.

Reply:
done